bundles/org.openhab.binding.souliss/README.md

```diff
@@ -0,0 +1,352 @@
+|RGB LED Strip|T16|souliss:t16|
+|ON/OFF Digital Output|T18|souliss:t18|
+|Single Color LED Strip|T19|souliss:t19|
+|Digital Input Pass Through|T1A|souliss:t1A|
+|Motorized devices with limit switches|T21|souliss:t21|
+|Motorized devices with limit switches and middle position|T22|souliss:t22|
+|Temperature control|T31|souliss:t31|
+|Anti-theft integration (Main)|T41|souliss:t41|
+|Anti-theft integration (Peer)|T42|souliss:t42|
+|Analog input, half-precision floating point|T51|souliss:t51|
+|Temperature measure (-20, +50) °C|T52|souliss:t52|
+|Humidity measure (0, 100) %|T53|souliss:t53|
+|Light Sensor (0, 40) kLux|T54|souliss:t54|
+|Voltage (0, 400) V|T55|souliss:t55|
+|Current (0, 25)  A|T56|souliss:t56|
+|Power (0, 6500)  W|T57|souliss:t57|
+|Pressure measure (0, 1500) hPa|T58|souliss:t58|
+|Analog Setpoint|T61|souliss:t61|
+|Analog Setpoint-Temperature measure (-20, +50) °C|T62|souliss:t62|
+|Analog Setpoint-Humidity measure (0, 100) %|T63|souliss:t63|
+|Analog Setpoint-Light Sensor (0, 40) kLux|T64|souliss:t64|
+|Analog Setpoint-Voltage (0, 400) V|T65|souliss:t65|
+|Analog Setpoint-Current (0, 25)  A|T66|souliss:t66|
+|Analog Setpoint-Power (0, 6500)  W|T67|souliss:t67|
+|Analog Setpoint-Pressure measure (0, 1500) hPa|T68|souliss:t68|
+|Broadcast messages|Action Message|souliss:topic|
+
+### Channels
+
+The following matrix lists the capabilities (channels) for each type:
+
+|Thing type / Channel |Switch / onoff | Switch / sleep | DateTime / lastStatusStored | Number / healthy |Switch / automode|Contact / stateOnOff|Contact / stateOpenClose|Switch / pulse|Switch / whitemode|Rollershutter / roller_brightness|Dimmer / dimmer_brightness|Color / ledcolor|Switch / one|Switch / two|Switch / three|Switch / four|Switch / five|Switch / six|Switch / seven|Switch / eight|
+|-- |-- | -- | -- | -- |--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|--|
+|souliss:t11|x|x|x|x||||
+|souliss:t12|x| |x|x|x||||
+|souliss:t13|||x|x||x|x||
+|souliss:t14|||x|x||||x|
+|souliss:t16|x|x|x|x|||||x|x|x|x||
+|souliss:t18|x||x|x|||||||||
+|souliss:t19|x|x|x|x||||||x|x|
+|souliss:t1A|||x|x|||||||||x|x|x|x|x|x|x|x|
+
+|Thing type / Channel | DateTime / lastStatusStored | Number / healthy |Rollershutter / rollershutter|(see down) / rollershutterstate|(see down) / mode|(see down) / fan|Switch / status|Number / setpoint|Switch / setAsMeasured|Switch / measured|Switch / statusAlarm|Switch / onOffAlarm|Switch / rearmAlarm|
+|-- |-- | --|- | -- | -- |--|--|--|--|--|--|--|--|
+|souliss:t21|x|x||x|
+|souliss:t22|x|x|x|x|
+|souliss:t31|x|x|||x|x|x|x|x|x|
+|souliss:t41|x|x|||||||||x|x|x|
+|souliss:t42|x|x|||||||||x||x|
+
+rollershutterstate = opening, closing, limSwitch_open , limSwitch_close, state_open, state_close, NoLimSwitch
+
+mode = COOLING_MODE, HEATING_MODE, POWEREDOFF_MODE
+
+fan = AUTO, HIGH, MEDIUM, LOW, FANOFF
+
+
+|Thing type / Channel| DateTime / lastStatusStored |Number / healthy|Number / value|
+|-- |-- | -- | --|
+|souliss:t51|x|x|x|
+|souliss:t52|x|x|x|
```

Comment:
This table does not make anything clearer to me, I dont understand what it is trying to show?
Typically people use this section of the readme to learn what channels a thing type has and what they do and the type of channel is it so they know how to setup the thing with the correct type of controls.

Take a look at how other bindings display the data in a table and what headings they have.

Reply:
cutted souliss before :  
Normally who uses Souliss understand the table.